### ISSUE TITLE
Keybindings Editor: Enter edit mode on Enter

### ISSUE THREAD

**Issue by kieferrm** [Microsoft/vscode#31832]
While testing #31784.

- VSCode Version: Code - Insiders 1.15.0-insider (f39385b8326671dc6389f21f8c2e2f6afdb352a1, 2017-08-01T05:07:59.267Z)
- OS Version: Darwin x64 16.7.0

---

Steps to Reproduce:

1. Select `Preferences: Open Keyboard Shortcuts` from the command palette
2. Type `switch `
3. <kbd>ArrowDown</kbd> to `Switch Terminal`
4. <kbd>Enter</kbd>
-> The row has now the keyboard focus but it should have entered edit mode directly. I still can get into edit mode by moving the keyboard focus with Shift+Tab, but it would be way more convenient to enter edit mode directly.
